AppCenter/AppCenterSDK+HiDrive.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  AppCenterSDK+SMFLogger.swift
+//  SmartMobileFactory
+//
+//  Created by Konstantin Deichmann on 11.09.19.
+//  Copyright © 2019 SmartMobileFactory. All rights reserved.
+//
+
+import Foundation
+import AppCenter
+import AppCenterCrashes
+import AppCenterDistribute
+import SMFLogger
+
+
+fileprivate enum AppCenterConstants {
+
+	static let appSecretKey					= "AppCenterAppSecret"
+	static let smfLogUploadMaxSize	: Int	= 5000
+}
+
+class AppCenterSDK: NSObject {
+
+	// MARK: - Private static properties
+
+	fileprivate static let isDebugBuild		: Bool = {
+		#if DEBUG
+		return true
+		#else
+		return false
+		#endif
+	}()
+
+	// MARK: - Private properties
+
+	fileprivate static var shared			: AppCenterSDK?
+	fileprivate var configuration			: Configuration?
```

Comment:
@konDeichmann: `self.configuration` is never set (this is also the case for the `AppCenterSDK+SMFLogger`).
Do we need to keep a reference to the configuration ?